### DISCUSSION TITLE
SBUS on 2.4Ghz rx

### DISCUSSION THREAD

**Discussion by rizenfpv** [ExpressLRS/ExpressLRS#531] [Ideas]
I have a Zohd KoPilot running on SBUS so i wondered if it was possible to create a 2.4Ghz SBUS receiver.

**Comment by kubrick89** [comment]
This would be really useful... Having SBUS output on rx would allow me to use an SBUS > PWM converter to control servos

**Comment by microraptor** [comment]
Edit:
Sounds like SBUS support would be quite a lot of work and since it is a legacy protocol, there are no plans to implement it: #1182 

Old anwser:
~SBUS is in development and discussed in #581:~
https://github.com/ExpressLRS/ExpressLRS/issues/581#issuecomment-865076299
